docs/docs/setting-up/jobs/job-specification/job.md

```diff
@@ -27,21 +29,29 @@ Tasks:
-- **Labels** <code>(<a href="./label">Label</a>[] : nil)</code>: Arbitrary labels associated with the job for filtering purposes.
-- **Constraints** <code>(<a href="./constraint">Constraint</a>[] : nil)</code>: These are selectors which must be true for a compute node to run this job.
-- **Tasks** <code>(<a href="./task">Task</a>[] : \<required\>)</code>:: Task associated with the job, which defines a unit of work within the job. Today we are only supporting single task per job, but with future plans to extend this. 
+- **Priority** `(int: 0)`: Determines the scheduling priority.
+- **Count** `(int: <required>)`: Number of replicas to be scheduled. This is only applicable for jobs of type `batch` and `service`.
+- **Meta** <code>(<a href="../job-specification/meta">Meta[]</a>: nil)</code>: Arbitrary metadata associated with the job.
+- **Labels** <code>(<a href="./label">Label[]</a>: nil)</code>: Arbitrary labels associated with the job for filtering purposes.
+- **Constraints** <code>(<a href="./constraint">Constraint[]</a>: nil)</code>: These are selectors which must be true for a compute node to run this job.
+- **Tasks** <code>(<a href="./task">Task[]</a>: \<required>\)</code>: Task associated with the job, which defines a unit of work within the job. Today we are only supporting single task per job, but with future plans to extend this.
+- **InputSource** <code>(<a href="./input-source">InputSource[]</a>: nil)</code>: Defines where and how to retreive specific artifacts needed for a Task, such as files or data.
+- **Network** <code>(<a href="./network">Network[]</a>: \<optional>\)</code>: Defines the scope and constraints of the network connectivity based on the demands of the Task.
+- **Resources** <code>(<a href="./resources">Resources[]</a>: \<optional>\)</code>: Provides a structured way to detail the computational resources a Task requires.
+- **ResultPath** <code>(<a href="./result-path">ResultPath[]</a>: \<optional>\)</code>: Denotes a specific location within a Task, that should be retained or published after Task's execution.
+- **SpecConfig** <code>(<a href="./spec-config">SpecConfig[]</a>: \<optional>\)</code>: Provides a unified structure to specify configurations for various components in Bacalhau, including engines, publishers, and input sources.
+- **Timeouts** <code>(<a href="./timeouts">Timeouts[]</a>: \<optional>\)</code>: Provides a mechanism to impose timing constraints on specific task operations, particularly execution.
 
 ## Server-Generated Parameters
 The following parameters are generated by the server and should not be set directly.
 - **ID** `(string)`: A unique identifier assigned to this job. It's auto-generated by the server and should not be set directly. Used for distinguishing between jobs with similar names.
-- **State** <code>(<a href="./state">State</a>)</code>: Represents the current state of the job.
+- **State** <code>(<a href="./state">State</a>)</code>resents the current state of the job.
```

Comment:
I think Represents was the correct word here, resents doesn't make a lot of sense.

Reply:
fixed